tests/issues.targets

```diff
@@ -391,6 +391,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/Interop/NativeLibraryResolveEvent/ResolveEventTests/*">
             <Issue>21964</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/Interop/NativeLibraryResolveCallBack/CallBackTests/*">
```

Comment:
I know this is a huge pain, but we should make this consistent. Please update these paths so they are cased correctly - `NativeLibraryResolveCallback` (i.e. callback is one word).

Reply:
Throughout.

Reply:
I'll update the name of this test. But there are a lot of places in the source code, and many tests that use the CallBack notation. I don't want to touch all that in this checkin (~2000 changes), can do it separately.

Reply:
@swaroop-sridhar Please don't take this as do it everywhere in the repo. Definitely don't want you to do that. I just don't want to add any more.